src/components/ComposedModal/ComposedModal.js

```diff
@@ -74,11 +74,12 @@ export default class ComposedModal extends Component {
 
     return (
       <div
+        {...other}
         role="presentation"
         ref={modal => (this.modal = modal)}
         onKeyDown={this.handleKeyDown}
         className={modalClass}
-        {...other}>
+        tabIndex={-1}>
```

Comment:
Would this contradict the `role="presentation"` prop set beforehand?

Reply:
it contradict the onKeyDown function